src/formio/templates/file.ejs

```diff
@@ -17,7 +17,11 @@
       <li class="list-group-item">
         <div class="row">
           {% if (!ctx.disabled) { %}
-            <div class="col-md-1"><i class="{{ctx.iconClass('remove')}}" ref="removeLink"></i></div>
+            <div class="col-md-1">
+              <button type="button" class="{{ctx.ofPrefix}}button {{ctx.ofPrefix}}button--invisible" ref="removeLink">
+                <i class="{{ctx.iconClass('remove')}}"></i>
+              </button>
+            </div>
```

Comment:
Can't we make this an `a` tag instead? Semantically that makes more sense and we then wouldn't need an additional variant of the button.

Reply:
We could. I just followed what Formio does in their template (the removal icons are inside buttons)

Reply:
After discussion: 
- Looked at both the [button that looks like a link with icon](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-link-button--with-icon) and the [button with icon](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-button--icon-after) but they don't really seem like what we need here?